android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java

```diff
@@ -51,7 +50,7 @@ private static void initPointerProps(int size) {
   private View mView;
   private int mState = STATE_UNDETERMINED;
   private float mX, mY;
-  private boolean mWithinBounds;
+  protected boolean mWithinBounds;
```

Comment:
This does not seem very consistent. We change visibility of this member only so that we can add public getter to it in a subclass? Why can't we do the same as we do in all the other variables around `mX`, `mY` etc and just have a getter directly in this class?